### ISSUE TITLE
Crash Report - Chunk Loading / Saving?

### ISSUE THREAD

**Issue by OriGummy** [PaperSpigot/Paper#7074]
### Stack trace

------------------------------
[13:03:43 ERROR]: The server has stopped responding! This is (probably) not a Paper bug.
[13:03:43 ERROR]: If you see a plugin in the Server thread dump below, then please report it to that author
[13:03:43 ERROR]:        *Especially* if it looks like HTTP or MySQL operations are occurring
[13:03:43 ERROR]: If you see a world save or edit, then it means you did far more than your server can handle at once
[13:03:43 ERROR]:        If this is the case, consider increasing timeout-time in spigot.yml but note that this will replace the crash with LARGE lag spikes
[13:03:43 ERROR]: If you are unsure or still think this is a Paper bug, please report this to https://github.com/PaperMC/Paper/issues
[13:03:43 ERROR]: Be sure to include ALL relevant console errors and Minecraft crash reports
[13:03:43 ERROR]: Paper version: git-Paper-19 (MC: 1.18)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Server thread dump (Look for plugins here before reporting to Paper!):
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Server thread
[13:03:43 ERROR]:       PID: 25 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               net.minecraft.server.level.ServerChunkCache.tickChunks(ServerChunkCache.java:1010)
[13:03:43 ERROR]:               net.minecraft.server.level.ServerChunkCache.tick(ServerChunkCache.java:923)
[13:03:43 ERROR]:               net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:622)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1606)
[13:03:43 ERROR]:               net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:480)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1469)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1263)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316)
>
Exception: java.lang.ThreadDeath thrown from the UncaughtExceptionHandler in thread "Server thread"
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer$$Lambda$3934/0x0000000801438240.run(Unknown Source)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Entire Thread Dump:
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Reference Handler
[13:03:43 ERROR]:       PID: 2 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.Reference.processPendingReferences(Reference.java:253)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.Reference$ReferenceHandler.run(Reference.java:215)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Finalizer
[13:03:43 ERROR]:       PID: 3 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Object.wait(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:172)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Signal Dispatcher
[13:03:43 ERROR]:       PID: 4 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Attach Listener
[13:03:43 ERROR]:       PID: 5 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Notification Thread
[13:03:43 ERROR]:       PID: 12 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Common-Cleaner
[13:03:43 ERROR]:       PID: 13 | Suspended: false | Native: false | State: TIMED_WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Object.wait(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:162)
> [13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: DestroyJavaVM
[13:03:43 ERROR]:       PID: 15 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: WindowsStreamPump
[13:03:43 ERROR]:       PID: 20 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               org.fusesource.jansi.internal.Kernel32.WaitForSingleObject(Native Method)
[13:03:43 ERROR]:               org.jline.terminal.impl.jansi.win.JansiWinSysTerminal.processConsoleInput(JansiWinSysTerminal.java:138)
[13:03:43 ERROR]:               org.jline.terminal.impl.AbstractWindowsTerminal.pump(AbstractWindowsTerminal.java:460)
[13:03:43 ERROR]:               org.jline.terminal.impl.AbstractWindowsTerminal$$Lambda$103/0x0000000800d1bce8.run(Unknown Source)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Log4j2-AsyncAppenderEventDispatcher-1-Async
[13:03:43 ERROR]:       PID: 22 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Thread is waiting on monitor(s):
[13:03:43 ERROR]:               Locked on:net.minecrell.terminalconsole.TerminalConsoleAppender.print(TerminalConsoleAppender.java:263)
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               org.jline.reader.impl.LineReaderImpl.redisplay(LineReaderImpl.java:3847)
[13:03:43 ERROR]:               org.jline.reader.impl.LineReaderImpl.printAbove(LineReaderImpl.java:778)
[13:03:43 ERROR]:               net.minecrell.terminalconsole.TerminalConsoleAppender.print(TerminalConsoleAppender.java:263)
[13:03:43 ERROR]:               net.minecrell.terminalconsole.TerminalConsoleAppender.append(TerminalConsoleAppender.java:256)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:156)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:129)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:120)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:84)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.appender.rewrite.RewriteAppender.append(RewriteAppender.java:84)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:156)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:129)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:120)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:84)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.appender.rewrite.RewriteAppender.append(RewriteAppender.java:84)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.tryCallAppender(AppenderControl.java:156)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppender0(AppenderControl.java:129)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppenderPreventRecursion(AppenderControl.java:120)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.config.AppenderControl.callAppender(AppenderControl.java:84)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.dispatch(AsyncAppenderEventDispatcher.java:119)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.dispatchAll(AsyncAppenderEventDispatcher.java:81)
[13:03:43 ERROR]:               org.apache.logging.log4j.core.appender.AsyncAppenderEventDispatcher.run(AsyncAppenderEventDispatcher.java:63)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Timer hack thread
[13:03:43 ERROR]:       PID: 23 | Suspended: false | Native: false | State: TIMED_WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.sleep(Native Method)
[13:03:43 ERROR]:               net.minecraft.Util$6.run(Util.java:651)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Worker-Main-1
[13:03:43 ERROR]:       PID: 24 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Java2D Disposer
[13:03:43 ERROR]:       PID: 27 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Object.wait(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
[13:03:43 ERROR]:               java.desktop@17.0.1/sun.java2d.Disposer.run(Disposer.java:145)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: AWT-Shutdown
[13:03:43 ERROR]:       PID: 28 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Object.wait(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Object.wait(Object.java:338)
[13:03:43 ERROR]:               java.desktop@17.0.1/sun.awt.AWTAutoShutdown.run(AWTAutoShutdown.java:291)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: AWT-Windows
[13:03:43 ERROR]:       PID: 29 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.desktop@17.0.1/sun.awt.windows.WToolkit.eventLoop(Native Method)
[13:03:43 ERROR]:               java.desktop@17.0.1/sun.awt.windows.WToolkit.run(WToolkit.java:365)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: AWT-EventQueue-0
[13:03:43 ERROR]:       PID: 31 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.text.AbstractDocument$AbstractElement.<init>(AbstractDocument.java:1816)
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.text.AbstractDocument$LeafElement.<init>(AbstractDocument.java:2549)
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.text.AbstractDocument.createLeafElement(AbstractDocument.java:1309)
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.text.PlainDocument.insertUpdate(PlainDocument.java:201)
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.text.AbstractDocument.handleInsertString(AbstractDocument.java:777)
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.text.AbstractDocument.insertString(AbstractDocument.java:739)
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.text.PlainDocument.insertString(PlainDocument.java:131)
[13:03:43 ERROR]:               net.minecraft.server.gui.MinecraftServerGui.print(MinecraftServerGui.java:199)
[13:03:43 ERROR]:               net.minecraft.server.gui.MinecraftServerGui.lambda$print$2(MinecraftServerGui.java:187)
[13:03:43 ERROR]:               net.minecraft.server.gui.ServerGUI$$Lambda$4044/0x000000080150bb80.run(Unknown Source)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventQueue$4.run(EventQueue.java:722)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventQueue$4.run(EventQueue.java:716)
[13:03:43 ERROR]:               java.base@17.0.1/java.security.AccessController.executePrivileged(AccessController.java:776)
[13:03:43 ERROR]:               java.base@17.0.1/java.security.AccessController.doPrivileged(AccessController.java:399)
[13:03:43 ERROR]:               java.base@17.0.1/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
[13:03:43 ERROR]:               java.desktop@17.0.1/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: TimerQueue
[13:03:43 ERROR]:       PID: 33 | Suspended: false | Native: false | State: TIMED_WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.DelayQueue.take(DelayQueue.java:229)
[13:03:43 ERROR]:               java.desktop@17.0.1/javax.swing.TimerQueue.run(TimerQueue.java:172)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Thread-1
[13:03:43 ERROR]:       PID: 32 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.LinkedBlockingQueue.take(LinkedBlockingQueue.java:435)
[13:03:43 ERROR]:               com.mojang.util.QueueLogAppender.getNextLogEvent(QueueLogAppender.java:77)
[13:03:43 ERROR]:               net.minecraft.server.gui.MinecraftServerGui.lambda$buildChatPanel$1(MinecraftServerGui.java:158)
[13:03:43 ERROR]:               net.minecraft.server.gui.ServerGUI$$Lambda$4030/0x000000080143f870.run(Unknown Source)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Server thread
[13:03:43 ERROR]:       PID: 25 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               net.minecraft.world.level.SpawnerCreature$$Lambda$4810/0x000000080186f500.accept(Unknown Source)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.Optional.ifPresent(Optional.java:178)
[13:03:43 ERROR]:               net.minecraft.server.level.ServerChunkCache.getFullChunk(ServerChunkCache.java:1081)
[13:03:43 ERROR]:               net.minecraft.server.level.ChunkProviderServer$$Lambda$4538/0x0000000801751028.query(Unknown Source)
[13:03:43 ERROR]:               net.minecraft.world.level.NaturalSpawner.createState(NaturalSpawner.java:103)
[13:03:43 ERROR]:               net.minecraft.server.level.ServerChunkCache.tickChunks(ServerChunkCache.java:995)
[13:03:43 ERROR]:               net.minecraft.server.level.ServerChunkCache.tick(ServerChunkCache.java:923)
[13:03:43 ERROR]:               net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:622)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1606)
[13:03:43 ERROR]:               net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:480)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1469)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1263)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316)
[13:03:43 ERROR]:               net.minecraft.server.MinecraftServer$$Lambda$3934/0x0000000801438240.run(Unknown Source)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Paper Async Chunk Task Thread #0
[13:03:43 ERROR]:       PID: 41 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
[13:03:43 ERROR]:               com.destroystokyo.paper.io.QueueExecutorThread.run(QueueExecutorThread.java:84)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Paper Async Chunk Urgent Task Thread
[13:03:43 ERROR]:       PID: 42 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
[13:03:43 ERROR]:               com.destroystokyo.paper.io.QueueExecutorThread.run(QueueExecutorThread.java:84)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Paper Watchdog Thread
[13:03:43 ERROR]:       PID: 43 | Suspended: false | Native: false | State: RUNNABLE
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.management@17.0.1/sun.management.ThreadImpl.dumpThreads0(Native Method)
[13:03:43 ERROR]:               java.management@17.0.1/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:521)
[13:03:43 ERROR]:               java.management@17.0.1/sun.management.ThreadImpl.dumpAllThreads(ThreadImpl.java:509)
[13:03:43 ERROR]:               org.spigotmc.WatchdogThread.run(WatchdogThread.java:204)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Server console handler
[13:03:43 ERROR]:       PID: 36 | Suspended: false | Native: false | State: TIMED_WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1757)
[13:03:43 ERROR]:               org.jline.utils.NonBlockingPumpReader.read(NonBlockingPumpReader.java:78)
[13:03:43 ERROR]:               org.jline.utils.NonBlockingReader.read(NonBlockingReader.java:57)
[13:03:43 ERROR]:               org.jline.keymap.BindingReader.readCharacter(BindingReader.java:160)
[13:03:43 ERROR]:               org.jline.keymap.BindingReader.readBinding(BindingReader.java:110)
[13:03:43 ERROR]:               org.jline.keymap.BindingReader.readBinding(BindingReader.java:61)
[13:03:43 ERROR]:               org.jline.reader.impl.LineReaderImpl.doReadBinding(LineReaderImpl.java:923)
[13:03:43 ERROR]:               org.jline.reader.impl.LineReaderImpl.readBinding(LineReaderImpl.java:956)
[13:03:43 ERROR]:               org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:651)
[13:03:43 ERROR]:               org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:468)
[13:03:43 ERROR]:               net.minecrell.terminalconsole.SimpleTerminalConsole.readCommands(SimpleTerminalConsole.java:158)
[13:03:43 ERROR]:               net.minecrell.terminalconsole.SimpleTerminalConsole.start(SimpleTerminalConsole.java:141)
[13:03:43 ERROR]:               net.minecraft.server.dedicated.DedicatedServer$1.run(DedicatedServer.java:111)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: pool-4-thread-1
[13:03:43 ERROR]:       PID: 44 | Suspended: false | Native: false | State: TIMED_WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Netty Server IO #0
[13:03:43 ERROR]:       PID: 45 | Suspended: false | Native: true | State: RUNNABLE
[13:03:43 ERROR]:       Thread is waiting on monitor(s):
[13:03:43 ERROR]:               Locked on:java.base@17.0.1/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
[13:03:43 ERROR]:               Locked on:java.base@17.0.1/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/sun.nio.ch.WEPoll.wait(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/sun.nio.ch.WEPollSelectorImpl.doSelect(WEPollSelectorImpl.java:111)
[13:03:43 ERROR]:               java.base@17.0.1/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
[13:03:43 ERROR]:               java.base@17.0.1/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:146)
[13:03:43 ERROR]:               io.netty.channel.nio.SelectedSelectionKeySetSelector.select(SelectedSelectionKeySetSelector.java:68)
[13:03:43 ERROR]:               io.netty.channel.nio.NioEventLoop.select(NioEventLoop.java:810)
[13:03:43 ERROR]:               io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:457)
[13:03:43 ERROR]:               io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
[13:03:43 ERROR]:               io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Thread.run(Thread.java:833)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Paper RegionFile IO Thread
[13:03:43 ERROR]:       PID: 46 | Suspended: false | Native: false | State: WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/jdk.internal.misc.Unsafe.park(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
[13:03:43 ERROR]:               com.destroystokyo.paper.io.QueueExecutorThread.run(QueueExecutorThread.java:84)
[13:03:43 ERROR]: ------------------------------
[13:03:43 ERROR]: Current Thread: Timer-0
[13:03:43 ERROR]:       PID: 47 | Suspended: false | Native: false | State: TIMED_WAITING
[13:03:43 ERROR]:       Stack:
[13:03:43 ERROR]:               java.base@17.0.1/java.lang.Object.wait(Native Method)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.TimerThread.mainLoop(Timer.java:563)
[13:03:43 ERROR]:               java.base@17.0.1/java.util.TimerThread.run(Timer.java:516)
[13:03:43 ERROR]: ------------------------------

### Plugin and Datapack List

Plugins:
Graves-3.8-fix-async-chunk-getEntities-call
OnePlayerSleep-3.1.7
GummyNode (An extremely simple chat plugin I created)

Datapacks:
None

### Paper version

§fThis server is running Paper version git-Paper-61 (MC: 1.18) (Implementing API version 1.18-R0.1-SNAPSHOT) (Git: 0e91b6a)§r
§eYou are 5 version(s) behind
Download the new version at: §6https://papermc.io/downloads§r
§7§oPrevious version: git-Paper-19 (MC: 1.18)

**Comment by jpenilla** [comment]
> git-Paper-19

update

Also, this seems better suited for asking for help in discord than an issue, but do feel free to open a new issue if this persists on the latest Paper.
